.github/workflows/deploy_appengine_test.yml

```diff
@@ -0,0 +1,47 @@
+name: Deploy to app engine
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  PROJECT_ID: dft-rlg-atip-test
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          cache: 'npm'
+
+      - name: Install wasm-pack
+        uses: jetli/wasm-pack-action@v0.4.0
+
+      - name: Build
+        run: |
+          npm ci
+          npm run setup-govuk
+          VITE_RESOURCE_BASE="https://${PROJECT_ID}.ew.r.appspot.com/data" npm run build
+          cd backend
+          rm -rf dist
+          cp -R ../dist .
+
+      - name: Authenticate to GCP
+        uses: google-github-actions/auth@v1
+        with:
+          credentials_json: '${{ secrets.GCP_TEST_CREDENTIALS }}'
+
+      - name: Deploy
+        uses: google-github-actions/deploy-appengine@v1
+        with:
+          project_id: ${{ vars.PROJECT_ID }}
+          deliverables: backend/app.yaml
```

Comment:
I think we'll need another copy of this file, https://github.com/acteng/atip/blob/8dc01575ddd1f6482eae9a88eb9a317a6e648a82/backend/app.yaml#L5

And it'd be good to rename the existing .yaml stuff to 'dev' to be clear about dev vs test